src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/config/AuthExceptionHandler.kt

```diff
@@ -12,8 +10,7 @@ import uk.gov.justice.digital.hmpps.manageusersapi.service.auth.NotFoundExceptio
 
 // Exception Handler to mirror the exceptions thrown by Auth.  Needed to ensure all endpoints moved from auth into this
 // service return the same exception detail.
-@RestControllerAdvice
-@Order(HIGHEST_PRECEDENCE)
+@RestControllerAdvice()
```

Comment:
Possible merge issue ?  We still need precedence

Reply:
Updated